core/state/state_object.go

```diff
@@ -238,6 +243,13 @@ func (self *StateObject) Nonce() uint64 {
 	return self.nonce
 }
 
+// Never called, but must be present to allow StateObject to be used
+// as a vm.Account interface that also satisfies the vm.ContractRef
+// interface. Interfaces are awesome.
+func (self *StateObject) Value() *big.Int {
+	return nil
```

Comment:
If this is never meant to be called, I think we should panic here (i.e. if someone does call it for some reason, fail hard with a stack dump so we can see something went very wrong).